### DISCUSSION TITLE
User profile settings/display template

### DISCUSSION THREAD

**Discussion by eswanzey** [vanjarosoftware/Vanjaro.Platform#1769] [Q&A]
How can the display of the User Profile be customized - is there a UI for that?

I removed some default user profile fields and added some new ones, but those changes aren't reflected in the displayed fields in the user's profile - they still show the original set of profile fields. I'm not sure this is a bug or roadmap item?

**Comment by meetmandeep** [comment]
Looks like missing functionality; moving this to issues as Bug.

**Comment by hshimanshusharma** [comment]
@eswanzey Make sure you are trying with "Registered User" or "Admin" not with SuperUser(host) because this field is not visible to SuperUser(host)
